_includes/js/queries.md

```diff
@@ -205,6 +205,39 @@ The above example will match any `BarbecueSauce` objects where the value in the
 
 Queries that have regular expression constraints are very expensive, especially for classes with over 100,000 records. Parse restricts how many such operations can be run on a particular app at any given time.
 
+### Full Text Search
+
+Use `fullText` for efficient search capabilities. Text indexes are automatically created for you. Your strings are turned into tokens for fast searching.
+
+Requires Parse-Server 2.5.0+
```

Comment:
So although Parse-Server is accurate to the project repo name we should probably be describing it as **Parse Server**. Unless we're referring to the repo itself the spaced name is a more appropriate way to describe it in most cases.

Reply:
I've typed parse-server more than I've typed my own name